pkg/kubeapiserver/options/authentication.go

```diff
@@ -756,7 +757,7 @@ func (o *BuiltInAuthenticationOptions) ApplyTo(
 					return
 				}
 
-				validationErrs := apiservervalidation.ValidateAuthenticationConfiguration(authConfig, authenticatorConfig.ServiceAccountIssuers)
+				validationErrs := apiservervalidation.ValidateAuthenticationConfiguration(authenticationcel.NewDefaultCompiler(), authConfig, authenticatorConfig.ServiceAccountIssuers)
```

Comment:
same question about constructing new or plumbing down ... not sure how to decide which to do here

Reply:
Applying options is a one-time operation. Hence, not creating a new one is fine.